src/Type.h

```diff
@@ -605,6 +605,12 @@ class RecordType final : public Type {
 	const TypeDecl* FieldDecl(int field) const;
 	TypeDecl* FieldDecl(int field);
 
+	// Returns flags corresponding to which fields in the record
+	// have types requiring memory management (reference counting).
+	// Primarily used by the compiler.
```

Comment:
I think that for newly added functions we should add doxygen-style comments for their usage.

This also is mentioned in https://github.com/zeek/zeek/wiki/Coding-Style-and-Conventions :)

This comment also applies to the following new functions.

Reply:
Have made a doxygen commetning pass.